script/github-actions/get-changed-apps.js

```diff
@@ -37,14 +35,15 @@ const getEntryName = filePath => {
 const getAllowedApp = (file, allowList, outputType = 'entry') => {
   if (!file.startsWith('src/applications')) return null;
 
-  const entryName = getEntryName(file);
+  const manifest = getManifest(file);
+  const entryName = manifest?.entryName;
 
   if (allowList.includes(entryName)) {
-    // Return app path when 'app-folders' option is used
+    // Return the app path, entry name, or root URL depending on the output type
     if (outputType === 'folder') {
       const appFolderName = file.split('/')[2];
       return `src/applications/${appFolderName}`;
-    }
+    } else if (outputType === 'url') return manifest?.rootUrl;
```

Comment:
Consider checking for all possible outputTypes and throwing if you reach the final `else` statement. That could help reduce future bugs

Reply:
Good point, I think that makes sense now that we're dealing with multiple output types. I just added that in there, along with a unit test for unknown output types ✅